tcl/Util.tcl

```diff
@@ -192,10 +194,14 @@ proc sta_warn { msg_id msg } {
 proc sta_error { msg_id msg } {
   variable sdc_file
   variable sdc_line
-  if { [info exists sdc_file] } {
-    error "Error: [file tail $sdc_file] line $sdc_line, $msg"
-  } else {
-    error "Error: $msg"
+  variable suppressed_msgs
+
+  if { ! [info exists suppressed_msgs] || ! $msg_id in $suppressed_msgs } {
+    if { [info exists sdc_file] } {
```

Comment:
use a function defined in Util.i to query instead of maintaining a separate list